.github/workflows/ci.yml

```diff
@@ -1,21 +1,24 @@
 name: CI
 on: [push, pull_request]
+
 jobs:
   ci:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Setup Ruby
-        uses: actions/setup-ruby@v1
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '3.0.3'
-      - run: npm install
+          # runs 'bundle install' and caches installed gems automatically
+          bundler-cache: true
+      - name: Use Node.js 16
```

Comment:
Is it worth adding a `.node-version` set to this version?

Reply:
👍